skyvern/forge/sdk/services/bitwarden.py

```diff
@@ -16,6 +17,11 @@
 LOG = structlog.get_logger()
 
 
+def is_valid_email(email: str) -> bool:
+    pattern = r"^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"
```

Comment:
is this regex good enough? how did you come up with this one?

Reply:
It's generated by GPT 😆, and I just check the logic here. 
Regex like email is a common case that you can find a good answer through the Internet. So GPT should be good.